docs/src/components/Typography/TypeScale.js

```diff
@@ -57,21 +57,18 @@ const formatSizes = sizes => {
 
 const getClassName = cssModules(STYLES);
 
-const TypeScale = props => {
-  const { className, ...rest } = props;
-
+const TypeScale = () => {
   return (
     <div
       aria-label="Type scale information"
-      className={getClassName('bpk-docs-type-scale', className)}
-      {...rest}
+      className={getClassName('bpk-docs-type-scale')}
     >
-      <div
-        aria-hidden="true"
-        className={getClassName('bpk-docs-type-scale__table')}
-      >
+      <div aria-hidden="true">
         {intervals.map(interval => (
-          <div className={getClassName('bpk-docs-type-scale__row')}>
+          <div
+            className={getClassName('bpk-docs-type-scale__row')}
+            key={interval.style}
```

Comment:
Added a key as there was a dev console error complaining that each component wasn't unique